src/NuGetGallery.Core/CredentialTypes.cs

```diff
@@ -25,6 +25,7 @@ public static class ApiKey
             public const string V2 = Prefix + "v2";
             public const string V3 = Prefix + "v3";
             public const string V4 = Prefix + "v4";
+            public const string V5 = Prefix + "v5";
```

Comment:
```suggestion
            public const string V5ShortLived = Prefix + "v5.shortlived";
```
Maybe we can call this `shortlived` for now. The V5 format using HISv2 (my draft PR I sent you) could be used for short lived or long-lived API keys in the future. Right now, we are focused on short lived which should be hidden from the UI and we can use the type string to differentiate between them. `v5.longlived` could be added later and may not be excluded from the UI.